supertokens_python/recipe/emailpassword/emaildelivery/services/backward_compatibility/__init__.py

```diff
@@ -41,16 +43,20 @@ def default_create_and_send_custom_email(
     async def func(user: User, password_reset_url_with_token: str, _: Dict[str, Any]):
         if ('SUPERTOKENS_ENV' in environ) and (environ['SUPERTOKENS_ENV'] == 'testing'):
             return
+        data = {}
         try:
             data = {
                 'email': user.email,
                 'appName': app_info.app_name,
                 'passwordResetURL': password_reset_url_with_token
             }
             async with AsyncClient() as client:
-                await client.post('https://api.supertokens.io/0/st/auth/password/reset', json=data, headers={'api-version': '0'})  # type: ignore
-        except Exception:
-            pass
+                resp = await client.post('https://api.supertokens.io/0/st/auth/password/reset', json=data, headers={'api-version': '0'})  # type: ignore
+                resp.raise_for_status()
```

Comment:
What does `resp.raise_for_status()` mean?

Reply:
If we get resp.status_code <= 599 or resp.status_code >= 400, it will raise an error (Like Axios in node)